Description: fix 4klang
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/music/4klang.inc
+++ b/music/4klang.inc
@@ -2,7 +2,7 @@
    global _%1
    _%1:
 %endmacro
-%define USE_SECTIONS
+;%define USE_SECTIONS
 %define SAMPLE_RATE	44100
 %define MAX_INSTRUMENTS	8
 %define MAX_VOICES 2
